content/soarian/dstu2/general-clinical/condition.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Condition | Soarian DSTU 2 API
+---
+ 
+# Condition
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Condition resource query, the API returns a consolidated list of conditions that providers have charted in the Soarian Problem List.  
+
+Problems returned may be in active, inactive, or resolved state.  Problems marked erroneous are not returned.  Both the problems that will be returned and the problem detail for each, depend on client configuration and charting practice. Problems returned include free text problems and those structured problems configured in the clinical care category.  The narrative response returns data as charted and may be more robust than the structured response. 
+
+
+## Terminology Bindings  
+
+<%= terminology_table(:soarian_condition, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
+| --- | --- | --- |
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list has no qualifying entries|**No relevant problems exist**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list contains qualifying entries|The following labels and values from the list will be returned in HTML, if valued: **Charted Problem Name, Assigned Date, Location, Status, Event, Day Part, Body Site, Frequency, Time Pattern, Intermittent, Rank, Severity, Body Location, Onset Date, Comment, Certainty, Clinical Care Category**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Problem list contains qualifying entries|Informational message that must be shown to the user: **This is the most current, known Problem list and may not reflect the state of problems known at a previous date.**|
+
+
+**CAUTION:** The detail that is in the narrative is not fully expressed in the structured portion of the FHIR response: The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  The structured component of the FHIR Condition resource response will primarily contain the code, code system, and code description for any coded problems in the list returned in the Narrative. This portion of the response may not be relevant to the user and should not be displayed. 
```

Comment:
"The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional""
This displays the entire text of the URL instead of a hyperlink. Change to
"the [narrative status(https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status) is "Additional""

Apply to diagnostic report as well